woocommerce-services.php

```diff
@@ -185,6 +185,17 @@ static function plugin_uninstall() {
 			WC_Connect_Options::delete_all_options();
 		}
 
+		/**
+		 * Get WCS plugin version
+		 *
+		 * @return string
+		 */
+		static function get_wcs_version() {
+			$file_path = plugin_dir_path( __FILE__ ) . 'woocommerce-services.php';
```

Comment:
**This** is the `woocommerce-services.php` file, you can just use `get_file_data(__FILE__, array(...))`

Reply:
Ah, right 😴